### DISCUSSION TITLE
[MongoDB Crud] Can I use the defaultSorting feature? How can I use it?

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#411] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

v12.2.1

### Description

Hi guys,
I read on the last release note about this interesting feature, [the crud-service `defaultSorting`](https://docs.mia-platform.eu/docs/release-notes/v12.2.1#new-features-3).

I need this feature for a client, but I am not understanding how I can use it.
I looked on the console but I didn't find anything, neither on the crud and endpoints section.

Is there a way to use this feature of the crud-service `v7.0.0`, with or without a UI?

Thanks.

**Comment by danibix95** [comment]
Hi @silversoul93, the feature will be soon configurable in Console, starting from version `v12.2.2`

**Reply by frengooo** [reply]
Hi @danibix95, thanks for your reply.
> starting from version v12.2.2

I'm happy to read that 😄 


Is there a way to manually use this feature for clients having the `v12.0.2` LTS installed?

**Reply by akelity** [reply, marked as answer]
Here the release note about defaultSorting: https://docs.mia-platform.eu/docs/12.x/release-notes/v12.3.0#default-sorting-index-for-crud-collections
